### PR TITLE
Add Michael Clark info and photo

### PR DESCRIPTION
**Pull request by mclarknc**
Add Michael Clark organizer info & photo

### CODE REVIEW

File: _organizers/michael-clark.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+github: jefftriplett
+layout: base
+name: Jeff Triplett
+photo_url: /static/img/organizers/jeff.jpg
+role: DEFNA Cofounder and President, Code of Conduct team
+mastodon: https://mastodon.social/@webology
+twitter: webology
+website: https://jefftriplett.com/
```
Comment (drewbrew) [inline _organizers/michael-clark.md:9]: I think you forgot to commit your changes here

Comment (mclarknc) [reply _organizers/michael-clark.md]: D'oh!  Just reinstalled Linux so I'm using VSCode rather than my usual PyCharm, which saves changes automatically.

Comment (jefftriplett) [reply _organizers/michael-clark.md]: Can confirm. Not @jefftriplett 😉

Comment (mclarknc) [reply _organizers/michael-clark.md]: But Jeff - I wanna be you!

Comment (jefftriplett) [reply _organizers/michael-clark.md]: Michael it's even more boring than it seems from afar. More meetings, though. ☕

**Review by drewbrew — APPROVED** [review]

**Pull request merged**